test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -421,6 +421,9 @@ $PendingCertificateTest = $false
 if ( $IsMacOS ) { $PendingCertificateTest = $true }
 if ( test-path /etc/centos-release ) { $PendingCertificateTest = $true }
 
+# Get the default UserAgent through reflection
+$DefaultUserAgent = [Microsoft.PowerShell.Commands.PSUserAgent].GetProperty('UserAgent',@('Static','NonPublic')).GetValue($null,$null)
```

Comment:
It is dummy test: if C# code will be changed the test will be automatically passed.  I believe we should use explicit desired pattern.
Also please move to `BeforeAll`.

Reply:
That's not the purpose of the tests. They are just testing the response from the server to ensure that expected data sent is also returned. if we want to test the patterns in the UserAgent, that would be a separate set of tests for the `PSUserAgent` class.

I will add a separate describe block for `PSUserAgent` tests and a test for the `App` member (there will be other tests possibly for #4193). I will create a separate issue to add and track adding other tests for `PSUserAgent`.